src/02_spec/07_modules/dem_uart/datainterface.rst

```diff
@@ -0,0 +1,15 @@
+Programmer Interface: Data
+--------------------------
+
+The Device Emulation Module only generates one type of event packet: **DEM UART Data packet**
+
+DEM UART Data packet
+^^^^^^^^^^^^^^^^^^^^
+
+A DEM UART Data packet contains one character, that has been sent to a UART interface
+by a CPU, as the only payload. Currently all Packets of this type are sent to the DII-Address ``0x00``.
```

Comment:
Please add a table showing the data format in detail, similar as in http://opensocdebug.readthedocs.io/en/latest/02_spec/07_modules/mam/datainterface.html

Sending all packets to address 0x00 is not ok, they need to sent to `MOD_EVENT_DEST`.